units/src/weight.rs

```diff
@@ -105,27 +105,27 @@ impl Weight {
 
     /// Checked addition.
     ///
-    /// Computes `self + rhs` returning `None` if an overflow occurred.
+    /// Computes `self + rhs` returning [`None`] if an overflow occurred.
     pub fn checked_add(self, rhs: Self) -> Option<Self> { self.0.checked_add(rhs.0).map(Self) }
 
     /// Checked subtraction.
     ///
-    /// Computes `self - rhs` returning `None` if an overflow occurred.
+    /// Computes `self - rhs` returning [`None`] if an overflow occurred.
     pub fn checked_sub(self, rhs: Self) -> Option<Self> { self.0.checked_sub(rhs.0).map(Self) }
 
     /// Checked multiplication.
     ///
-    /// Computes `self * rhs` returning `None` if an overflow occurred.
+    /// Computes `self * rhs` returning [`None`] if an overflow occurred.
     pub fn checked_mul(self, rhs: u64) -> Option<Self> { self.0.checked_mul(rhs).map(Self) }
 
     /// Checked division.
     ///
-    /// Computes `self / rhs` returning `None` if `rhs == 0`.
+    /// Computes `self / rhs` returning [`None`] if `rhs == 0`.
     pub fn checked_div(self, rhs: u64) -> Option<Self> { self.0.checked_div(rhs).map(Self) }
 
-    /// Scale by witness factor.
+    /// Scales by the witness factor.
```

Comment:
I prefer the original because the witness factor is not really a well known thing. I could be wrong, grammar is definitely not my strong suit.

Reply:
Removed "the"